Answers.md

```diff
@@ -1 +1,13 @@
+forEach : iterates over an array of objects which can be passed into a single argument function
 
+.map : map will create a new array from a function with new parameters passed into it.
+
+closure : is a function within a function in which the inner function has access to the variables created in the outer function. 
+
```

Comment:
Exactly what I was looking for. When you're asked to write examples, try to imagine you're explaining it to a 10 year old. If you can explain it to a 10 year old, you can understand the work.

Reply:
A good way to think about this that will help you understand this is:
"A closure is the combination of a function bundled together (enclosed) with references to its surrounding state (the lexical environment). In other words, a closure gives you access to an outer function’s scope from an inner function. In JavaScript, closures are created every time a function is created, at function creation time."